src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -446,6 +447,56 @@ public void CleanupAfterEachTest()
             }
         }
 
+        /// <summary>
+        /// Checks if the connection string provided in the config is correctly updated for MSSQL.
+        /// If the connection string already contains the `Application Name` property, it should update the existing value along with the DataApiBuilder Application Name.
+        /// If not, it should appends the property `Application Name` to the connection string.
+        /// </summary>
+        /// <param name="databaseType">database type.</param>
+        /// <param name="providedConnectionString">connection string provided in the config.</param>
+        /// <param name="expectedUpdatedConnectionString">Updated connection string with Application Name.</param>
+        /// <param name="isHostedScenario">If Dab is hosted or OSS.</param>
+        [DataTestMethod]
+        [DataRow(DatabaseType.MSSQL, "Something;", "Something;Application Name=dab_oss_1.0.0;", false, DisplayName = "[MSSQL]:Connection String doesn't contain Application name for DAB oss.")]
```

Comment:
```suggestion
        [DataRow(DatabaseType.MSSQL, "Something;", "Something;Application Name=dab_oss_1.0.0;", false, DisplayName = "[MSSQL]:Connection String contains Application name for DAB oss.")]
```

Reply:
it's correct, right? the connection string doesn't contain so we add Application name property